app/pages/components/HomePage.js

```diff
@@ -3,23 +3,86 @@
 import * as React from 'react';
 import { translate } from 'react-i18next';
 import type { TranslatorProps } from 'react-i18next';
-import { Link } from 'react-router-dom';
-import { Button } from 'semantic-ui-react';
+import { Feed, Grid } from 'semantic-ui-react';
+
+import Page from '../Page';
 
 type Props = TranslatorProps & { /* new props go here */ };
 
 const HomePage = (props: Props): React.Node => {
   const { t } = props;
 
   return (
-    <div>
-      <p><Link to="/topics">Topics page</Link></p>
-      <p>
-        <Button>{t('common:button.cancel')}</Button> {`<-- This doesn't do anything!`}
-      </p>
-    </div>
+    <Page>
+      <Grid.Row>
```

Comment:
This should be extracted into a separate 'List' (or 'Feed' or similar) component in the feed-items module and then included in the page.

Reply:
Is being worked on in a separate branch